.github/workflows/gh-pages-deployment.yml

```diff
@@ -45,4 +45,8 @@ jobs:
           publish_branch: deployment
 
           user_name: github-actions[bot]
-          user_email: 41898282+github-actions[bot]@users.noreply.github.com # default github-actions email
\ No newline at end of file
+          user_email: 41898282+github-actions[bot]@users.noreply.github.com
+          # The email used below is the default GitHub Action bot email.
```

Comment:
inconsistent indent in the block below. put the comment block above what you're commenting

Reply:
sorry about that, I indented it and changed placement, I am not sure if it's better to leave a blank like between user_email and start of comment for readability. I opted for not but if you think it's better to do so lmk